`math_tests__math__iree_vulkan__functions__atanh` failing on vulkan

Preemptively filing because disabling due to a failure in an upcoming change.

```
Diagnostics:
/usr/local/google/home/laurenzo/src/iree/iree.venv/lib/python3.9/site-packages/tensorflow/python/framework/op_def_library.py:748:0: error: failed to legalize operation 'math.log1p'
/usr/local/google/home/laurenzo/src/iree/iree.venv/lib/python3.9/site-packages/tensorflow/python/ops/gen_math_ops.py:1332:0: note: called from
/usr/local/google/home/laurenzo/src/iree/integrations/tensorflow/e2e/math/math_test.py:672:0: note: called from
/usr/local/google/home/laurenzo/src/iree/iree.venv/lib/python3.9/site-packages/tensorflow/python/autograph/core/function_wrappers.py:117:0: note: called from
/usr/local/google/home/laurenzo/src/iree/integrations/tensorflow/e2e/math/math_test.py:672:0: note: called from
/usr/local/google/home/laurenzo/src/iree/iree.venv/lib/python3.9/site-packages/tensorflow/python/framework/func_graph.py:983:0: note: called from
/usr/local/google/home/laurenzo/src/iree/iree.venv/lib/python3.9/site-packages/tensorflow/python/eager/def_function.py:668:0: note: called from
/usr/local/google/home/laurenzo/src/iree/iree.venv/lib/python3.9/site-packages/tensorflow/python/framework/func_graph.py:1007:0: note: called from
/usr/local/google/home/laurenzo/src/iree/iree.venv/lib/python3.9/site-packages/tensorflow/python/eager/function.py:3291:0: note: called from
/usr/local/google/home/laurenzo/src/iree/iree.venv/lib/python3.9/site-packages/tensorflow/python/eager/function.py:3456:0: note: called from
<unknown>:0: error: failed to run translation of source executable to target executable for backend vulkan*
<unknown>:0: error: Failures have been detected while processing an MLIR pass pipeline
<unknown>:0: note: Pipeline failed while executing [`mlir::iree_compiler::IREE::HAL::TranslateExecutablesPass` on 'hal.executable.target' operation: @vulkan_spirv, `mlir::iree_compiler::(anonymous namespace)::ConvertToSPIRVPass` on 'module' operation]: reproducer generated at `/usr/local/google/home/laurenzo/src/iree-build/integrations/tensorflow/e2e/math/math_tests__math__iree_vulkan__functions__atanh_test_tmpdir/tf/math/atanh/static_dims_non_complex/reproducer__iree_vulkan.mlir`
<unknown>:0: error: conversion from source -> vm failed
```

--- a/integrations/tensorflow/e2e/math/BUILD
+++ b/integrations/tensorflow/e2e/math/BUILD
@@ -292,7 +292,6 @@ VULKAN_FAILING = [
     "asinh",
     "atan",
     "atan2",
-    "atanh",
     "bessel_i0",
     "bessel_i0e",
     "bessel_i1",

--- a/integrations/tensorflow/e2e/math/CMakeLists.txt
+++ b/integrations/tensorflow/e2e/math/CMakeLists.txt
@@ -141,7 +141,6 @@ iree_e2e_cartesian_product_test_suite(
     ",,asinh,,,iree_vulkan"
     ",,atan,,,iree_vulkan"
     ",,atan2,,,iree_vulkan"
-    ",,atanh,,,iree_vulkan"
     ",,bessel_i0,,,iree_vulkan"
     ",,bessel_i0e,,,iree_vulkan"
     ",,bessel_i1,,,iree_vulkan"

